api/v1alpha1/oadp_types.go

```diff
@@ -233,7 +234,11 @@ type SnapshotLocation struct {
 }
 
 // Features defines the configuration for the DPA to enable the tech preview features
-type Features struct{}
+type Features struct {
+	// Enables non admin feature, by default false
+	// +optional
+	EnableNonAdmin *bool `json:"enableNonAdmin,omitempty"`
```

Comment:
Maybe we should introduce a `nonAdmin` struct here ?
If we need some more user input/customization data then in that scenario the `nonAdmin` feature struct would make a lot of sense. `Enable` bool could be one of the struct fields. Let me know what you think.

Reply:
will make the change!